.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: Documentation
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
```

Comment:
our branch is called `main` not `master`

I think this is the only necessary change.

Reply:
Good catch let me fix that real quick